tests/console/btrfs_qgroups.pm

```diff
@@ -31,8 +31,7 @@ my $dest = "/mnt/qg";
 # poo#11446
 sub run {
     my ($self) = @_;
-    select_console 'root-console';
-
+    $self->select_serial_terminal;
```

Comment:
This change is still not related to removing a soft-fail. It might be a valuable change but is probably better suited within a separate PR and should have proper testing on different backends and machines.